### PR TITLE
Update SDK to 26

### LINKED ISSUE #591: Laggy scroll
First off, great app!

I was looking at a user profile, and it seems to be somewhat laggy/jerky when scrolling. I think it might have to do with hiding the top fragment when scrolling down. 

Example, flicking to scroll won't work, it will just hide the top frightened and then sit there. Subsequent scrolling is fine.

### PR DESCRIPTION
**Pull request by Tunous**
Closes #591

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -27,11 +28,11 @@ android {
...
         playStore
     }
-    compileSdkVersion 24
-    buildToolsVersion '25.0.2'
+    compileSdkVersion 26
+    buildToolsVersion '26.0.1'
     defaultConfig {
         minSdkVersion 16
-        targetSdkVersion 23
+        targetSdkVersion 26
```
Comment (maniac103) [inline app/build.gradle]: Are we sure this won't have any side effects? Updating the target SDK version disables compatibility behavior for behavior changes in newer Android versions after all. Glancing over the behavior changes document I don't see any obvious breakage for us, but I'd like us to try first nevertheless ;-)

Comment (Tunous) [inline app/build.gradle]: Right, I've previously looked at this when updating SDK  to 25 for 
 comment bottom sheet. See https://github.com/slapperwan/gh4a/pull/536#issuecomment-290664355 to see my findings. In short, the biggest potential issue is where we pass a large amount of data through Intents as they have a hard limit now. But I don't have ideas how to fix that one....

I'll take another look later and also check for changes to 26 which wasn't released when I tested previously.

**Comment by Tunous** [comment]
Rebased onto master and reviewed [behavior changes](https://developer.android.com/about/versions/oreo/android-8.0-changes.html) and [migration guide](https://developer.android.com/about/versions/oreo/android-8.0-migration.html) for Android 8.0. Also tested on 8.0 emulator. Didn't find any issues.

**Comment by maniac103** [comment]
Thanks for checking!

**Comment by sharkykh** [comment]
Should the README section on how to build OctoDroid also be updated? (v24 => v26)

**Comment by Tunous** [comment]
Yeah, I'll fix that

**Pull request merged**